packages/twenty-front/src/modules/analytics/graphql/queries/track.ts

```diff
@@ -1,8 +1,13 @@
 import { gql } from '@apollo/client';
 
 export const TRACK = gql`
-  mutation Track($action: String!, $payload: JSON!) {
-    track(action: $action, payload: $payload) {
+  mutation Track(
```

Comment:
We have to be careful when rolling out this kind of changes that breaks the API called by the frontend, how do we make sure it doesn't trigger big errors for every person that hasn't yet refreshed their frontend? It frustrates me to spend time on this and slow us down at our stage, but on the other hand I don't see how we could circumvent it